ADR/0035-artifact-streams.md

```diff
@@ -0,0 +1,143 @@
+# 32. Decoupling Deployment
```

Comment:
copy and paste error?

Reply:
Yes, thanks. I had forgotten that I copied the previous ADR for structure.